disc-golf-blazor-ui/Services/AuthService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Net.Http.Json;
+
+namespace disc_golf_blazor_ui.Services
+{
+    public class AuthService
+    {
+        private readonly HttpClient httpClient;
+
+        public AuthService(HttpClient httpClient)
+        {
+            this.httpClient = httpClient;
+        }
+
+        public async Task<bool> LoginAsync(string email, string password)
+        {
+            var loginRequest = new LoginRequest
+            {
+                Email = email,
+                Password = password
+            };
+
+            var response = await httpClient.PostAsJsonAsync(
+                "http://localhost:24001/api/axion-auth-service/public/auth/login",
```

Comment:
This is interesting, you send request to localhost:24001, but you also define base URL for the same address localhost:24001 in other file:
BaseAddress = new Uri("https://localhost:24001")
I guess you don't need to specify localhost address in 2 places.

Reply:
Yes, it is better practice to use BaseAddress to avoid hard-coding full URLs. I will correct this soon.